source/features/clean-conversation-headers.css

```
@@ -1,9 +1,15 @@
 :root .rgh-clean-conversation-headers-non-default-branch {
-	background-color: var(--color-state-hover-primary-bg, #0366d6);
+	background-color: var(
+		--control-checked-bgColor-hover,
+		var(--color-state-hover-primary-bg, #0366d6)
+	);
 }
 
 .rgh-clean-conversation-headers-non-default-branch a {
-	color: var(--color-state-hover-primary-text, #fff);
+	color: var(
+		--control-checked-fgColor-rest,
```

I think this should match the `bicolor` state, which is

```suggestion
		--control-checked-fgColor-hover,
```